scripts/createTestApp.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { spawnPiped, waitFor, waitForExit, print } = require('./helpers');
+
+async function createTestApp() {
+  print('Creating a Backstage App');
+  const createApp = spawnPiped(['yarn', 'create-app']);
```

Comment:
We should create the app in a new temporary directory. Otherwise we might not spot issues with missing packages or git, since the new app might pick up things from this repo.

Reply:
👍 Yes, I suspected that we would want that...